api/mapping/views.py

```diff
@@ -253,6 +253,19 @@ class DatasetListView(generics.ListAPIView):
     # permission_classes = []
 
 
+class DatasetFilterView(generics.ListAPIView):
+    """
+    API view to filter datasets by list of id's.
+    """
+
+    queryset = Dataset.objects.all()
+    serializer_class = DatasetSerializer
+    filter_backends = [DjangoFilterBackend]
+    filterset_fields = {
+        "id": ["in"],
+    }
```

Comment:
could you also add in field for filtering by project. So if we wanted, we could restrict the listed datasets to those on a specific project

Reply:
why are we adding this as a `generics.ListAPIView` rather than in the class serialisers?

i.e. not here: https://github.com/CO-CONNECT/mapping-pipeline/blob/devel/api/mapping/serializers.py

To be honest, this type of view looks better/simpler than how we've got other api endpoints setup and working (which involved creating a view as well as a serialiser)

Reply:
Yeah I think that's right